Plan/common/src/main/java/com/djrapitops/plan/delivery/rendering/json/graphs/GraphJSONCreator.java

```diff
@@ -345,4 +345,14 @@ public String pingGraphsJSON(UUID serverUUID) {
                 .put("server_pie_series_30d", graphs.pie().serverPreferencePie(playtimePerServer).getSlices())
                 .build();
     }
+
+    public Map<String, Object> playerHostnamePieJSONAsMap() {
+        String[] pieColors = theme.getPieColors(ThemeVal.GRAPH_WORLD_PIE);
+        Map<String, Integer> hostnameResults = dbSystem.getDatabase().query(UserInfoQueries.hostnameTotals());
+
+        return Maps.builder(String.class, Object.class)
+                .put("server_pie_colors", pieColors)
+                .put("server_pie_series_30d", graphs.pie().hostnamePie(hostnameResults).getSlices())
```

Comment:
The name implies last 30 days but query does not have date as parameter to determine the time-span for the data